ward/terminal.py

```diff
@@ -370,7 +487,45 @@ def output_all_test_results(
         output_tests = self.runtime_output_strategies.get(
             self.test_output_style, output_test_per_line
         )
-        all_results = output_tests(fail_limit, test_results_gen)
+
+        spinner = SpinnerColumn(style="pass.textonly")
+        bar = BarColumn(complete_style="pass.textonly")
+        progress = Progress(
+            spinner,
+            TimeElapsedColumn(),
+            bar,
+            "[progress.percentage]{task.percentage:>3.0f}%",
+            "[progress.percentage][{task.completed} / {task.total}]",
+            TimeRemainingColumn(),
+            console=console,
+            transient=True,
+            auto_refresh=True,
+            # TODO: figure out how to make the progress bar work with printing with end="" (for dots)
+            disable=TestProgressStyle.BAR not in self.progress_styles
```

Comment:
The progress bar seems to always show for me as it stands

Reply:
This is because there was a bug in Rich that I fixed for v10 :) https://github.com/willmcgugan/rich/pull/1125

I'll update Ward's Rich version to 10.0.1, which should make it actually be disabled as expected.